federationexample/devicegqlserver/main.go

```diff
@@ -0,0 +1,92 @@
+
+
+package main
+
+import (
+	"context"
+	"log"
+	"net"
+	
+	"google.golang.org/grpc"
+
+	"github.com/samsarahq/thunder/federation"
+	"github.com/samsarahq/thunder/graphql/schemabuilder"
+	"github.com/samsarahq/thunder/thunderpb"
+
+)
+
+func schema() *schemabuilder.Schema {
+	schema := schemabuilder.NewSchemaWithName("device")
+
+
+	type Location struct {
+		Latitude float64
+		Longitude float64
+	}
+	location := schema.Object("Location", Location{}, 
+		schemabuilder.FetchObjectFromKeys(func(ctx context.Context, args struct{ Keys []*Location }) ([]*Location) {
+			return args.Keys
```

Comment:
It would be even better, if we can add a comment describing what these Keys are referring to, and how they would be used by this framework.